scripts/translator.js

```diff
@@ -0,0 +1,47 @@
+import morseCode from './morse-characters.js';
+
+// Keys are morse code, and values arecorresponding english
+const englishFromMorse = Object.entries(morseCode).reduce((acc, [key, value]) => {
+  acc[value] = key;
+  return acc;
+}, {});
+
+// English to morse code
+export function translateToMorse(english) {
+  const result = [];
+  // go over each character
+  for (let char of english.toUpperCase()) {
+    if (char === ' ') {
+      // Use "/" as for spaces in morse code
+      result.push('/');
+    } else if (morseCode[char]) {
+      // Translate character to morse code
+      result.push(morseCode[char]);
+    } else {
+      // error for unsupported characters
+      throw new Error(`${char} cannot be translated to Morse code.`);
+    }
```

Comment:
this throw does interrupt the whole translation, is this what you want? 
There are a couple other options if not